src/applicationManager.test.ts

```diff
@@ -157,11 +158,6 @@ describe("applicationManager", () => {
       optionalClaims: [{ name: "groups", additionalProperties: [] }],
     });
 
-    let application = await readApplication({
```

Comment:
the application  is re-retrived at line 176 and then referenced afterwards hence I removed this call